lib/membrane/rtp/vad.ex

```diff
@@ -121,6 +121,8 @@ defmodule Membrane.RTP.VAD do
 
   defp handle_if_present(buffer, nil, state), do: {{:ok, buffer: {:output, buffer}}, state}
 
+  @timestamp_limit 4_294_967_295
```

Comment:
Can you replace it with `Bitwise` once again? It is much easier to read when we use that :)